Feature/assinaturas-atualizar-json (#2412)
mercadopago/devsite-docs: guides/subscriptions/subscription-management.pt.md

| Tipo | Descrição |
|---|---|
| Buscar assinatura | Permite buscar as assinaturas independentemente do seu status (ativa, pausada, cancelada). Para isso, envie um GET com os parâmetros necessários ao endpoint [/preapproval/search](/developers/pt/reference/subscriptions/_preapproval_search/get) e execute a requisição. |

> | Alterar valor | Permite alterar o valor de uma assinatura existente. Envie o novo valor por meio de `auto_recurring.transaction_amount` e `auto_recurring.currency_id` em um PUT para o endpoint [/preapproval/{id}](/developers/pt/reference/subscriptions/_preapproval_id/put). |

Reviewer: ```suggestion
| Alterar valor | Permite alterar o valor de uma assinatura existente. Para isso, envie um PUT com os parâmetros `auto_recurring.transaction_amount` e `auto_recurring.currency_id`  ao endpoint [/preapproval/{id}](/developers/pt/reference/subscriptions/_preapproval_id/put) e execute a requisição.
```

> | Alterar valor | Permite alterar o valor de uma assinatura existente. Para isso, envie um PUT com os parâmetros `auto_recurring.transaction_amount` e `auto_recurring.currency_id`  ao endpoint [/preapproval/{id}](/developers/pt/reference/subscriptions/_preapproval_id/put) e execute a requisição.

Reviewer 2: ```suggestion
| Alterar valor | Permite alterar o valor de uma assinatura existente. Para isso, envie um PUT com os parâmetros `auto_recurring.transaction_amount`, especificando o novo valor a ser cobrado na assinatura, e `auto_recurring.currency_id`, indicando o identificador da moeda utilizada no pagamento, para o endpoint [/preapproval/{id}](/developers/pt/reference/subscriptions/_preapproval_id/put). Em seguida, execute a requisição.
```

> | Alterar cartão do meio de pagamento primário | Permite alterar o cartão associado à uma assinatura existente. Envie um PUT com o novo token no atributo `card_token_id` para o endpoint [/preapproval/{id}](/developers/pt/reference/subscriptions/_preapproval_id/put). |

Reviewer: ```suggestion
| Alterar cartão do meio de pagamento primário | Permite alterar o cartão associado à uma assinatura existente. Para isso, envie um PUT com o novo _token_ (no atributo `card_token_id`) ao endpoint [/preapproval/{id}](/developers/pt/reference/subscriptions/_preapproval_id/put) e execute a requisição. |
```

> |  Alterar meio de pagamento secundário | Permite adicionar um segundo meio de pagamento à uma assinatura existente. Se for um cartão, envie `card_token_id_secondary` e `payment_method_id_secondary`. Para outros métodos, apenas `payment_method_id_secondary`. Envie um PUT no endpoint [/preapproval/{id}](/developers/pt/reference/subscriptions/_preapproval_plan_id/put) para realizar a operação. |

Reviewer: ```suggestion
|  Alterar meio de pagamento secundário | Permite adicionar um segundo meio de pagamento à uma assinatura existente. Se for um cartão, envie um PUT com os parâmetros `card_token_id_secondary` e `payment_method_id_secondary` ao endpoint [/preapproval/{id}](/developers/pt/reference/subscriptions/_preapproval_plan_id/put) e execute a requisição. Para outros meios de pagamento, envie apenas o parâmetro `payment_method_id_secondary` e siga com a operação.  |
```
